src/pages/home/components/LoginBanner.tsx

```
@@ -7,44 +7,59 @@ import { useUserStore } from '@store/useUserStore';
           <MyDreamArrow />
         </div>
 
         <div className="absolute left-[30px] top-10 flex flex-col">
-          <span className="text-white font-B02-M"> 58일째 꿈꾸는중</span>
+          <span className="text-white font-B02-M">
+            {todoData ? `${todoData.daysAgo}일째 꿈꾸는중` : '꿈꾸는중'}
+          </span>
 
           <div className="mt-[10px] text-white font-T01-B">
-            요양보호사 시작하는 중
+            {todoData && todoData.jobName ? `${todoData.jobName} 시작하는 중` : '직업 준비중'}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)